universal-application-tool-0.0.1/app/services/program/ActiveAndDraftPrograms.java

```diff
@@ -0,0 +1,66 @@
+package services.program;
+
+import akka.japi.Pair;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import java.util.Optional;
+import models.Program;
+import models.Version;
+
+public class ActiveAndDraftPrograms {
```

Comment:
Suggestion (blocking): please document this class

Reply:
Done.